tutorials/workzone-enrich-5-integrate-card/workzone-enrich-5-integrate-card.md

```diff
@@ -39,13 +39,13 @@ You've already deployed a UI5 integration card. Now in this tutorial, you'll ena
 
     >Note: if you don't see your deployed card, you may need to refresh the browser.
 
-4. Click the slider button to enable the card and make it available for users who can then add it to Work Zone pages.
+4. Click the slider button to enable the card.
 
     <!-- border -->![Enable card](3-enable-card.png)
 
-The card is now enabled and can be added to your workspace.
+The card is now enabled and can be added to the workpage in your workspace.
 
->In this step you select your own card with your unique identifier `<your unique identifier>_Products by Vendor Card`.
+>In this step, for workshops, use your unique identifier `<your unique identifier>_Products by Vendor Card`.
 
 >Note: you can click on **Configure** to select a different destination for this UI5 integration card. As you used the same ES5 destination in this SAP BTP subaccount and also when you developed the card, this step is not required to complete this tutorial.
```

Comment:
Please don't use the label "note" for a note, because the formatting already makes it a note. If you want to emphasize the note, use labels like "important".

Reply:
Done